packages/interaction-tracking/src/InteractionZone.js

```diff
@@ -42,19 +62,42 @@ export function wrap(callback: Function): Function {
   }
 
   const wrappedContext = currentContext;
+  const id = ++executionID;
```

Comment:
What's the value in differentiating the execution id and the context/interaction id(s)? It's fine, I'm just curious what it lets us do.

Reply:
the execution ID allows us to pair up calls of onContextScheduled/Starting/Ended. Check out how InteractionProfiler is using the ID.